### DISCUSSION TITLE
corrupted version of apk - released

### DISCUSSION THREAD

**Discussion by vladut-ilie-ptt** [quasarframework/quasar#14182] [CLI - Capacitor / Cordova modes]
Hello, boyz.

I have an idiot question. I stared to develop an app using quasar, but when I trying to build the release version of the app, it can't be installed.
<img width="395" alt="image" src="https://user-images.githubusercontent.com/103733620/184355061-ba6068c6-7186-4adc-931f-b9fb0e838475.png">

If i will build the debug version, it's all gud. Any explanation for this ?

**Comment by metalsadman** [comment]
sign the apk or use `--debug` flag on your build command ie. `quasar build -m capacitor --debug`

**Reply by vladut-ilie-ptt** [reply]
Sorry for the late reply. 
The error after command `quasar build -m cordova -T android --debug`
<img width="1578" alt="image" src="https://user-images.githubusercontent.com/103733620/185762443-01beee9d-ed43-4f53-9953-1145ee4be220.png">

**Reply by metalsadman** [reply]
copy paste the error into search engine, it's saying you haven't installed a jdk.
